composer.json

```
@@ -214,7 +214,7 @@
         "enyo/dropzone": "5.7.2",
         "furf/jquery-ui-touch-punch": "dev-master",
         "geocoder-php/open-cage-provider": "^4.4",
-        "massgov/mayflower-artifacts": "11.11.0",
+        "massgov/mayflower-artifacts": "dev-DP-22653",
```

```suggestion
        "massgov/mayflower-artifacts": "11.11.0",
```
This should be changed to match the latest Mayflower version in develop